docs/persistence/README.md

```diff
@@ -0,0 +1,160 @@
+# Persistent Data
+
+In this section we will discuss how to work with persistent data in your
+applications and the different options available to you.
+
+Persistent data is data that is stored on disk and survives application
+restarts. This is in contrast to ephemeral data which is stored in memory
+and is lost when the application is restarted.
+
+## Responsibilities
+
+The team is reponsible for any data that is stored in the various storage
+options that are available through the platform. You can read more in the
+[Data Responsibilities](./responsibilities.md) section.
+
+## What should I choose?
+
+Sequence of questions to ask yourself when choosing the right storage option.
+Choose wisely.
+
+```mermaid
+graph TD
+  A[I got data!] --> B[Is it structured?]
+  B --> |Yes| C[Is it events?]
+  B --> |No| D[Is it files?]
+  C --> |Yes| Kafka
+  C --> |No| E[Is it analytical?]
+  D --> |Yes| GCS[Cloud Storage]
+  D --> |No| Opensearch[OpenSearch]
```

Comment:
OpenSearch kan fungere veldig bra på strukturerte data også. På grunn av hvor det kommer fra så er det lett å tenke på det som en søkemotor for "logg tekst", men det er egentlig en dokument-database med avansert søkefunksjonalitet.

Reply:
Ble fort veldig rotete, jeg lar denne ligge hvis noen andre vil plukke den opp senere.